Assets/_Functionalities/ObjectLibrary/Scripts/ObjectLibraryButton.cs

```diff
@@ -47,7 +48,7 @@ private void SpawnObject(Vector3 opticalSpawnPoint, bool hasHit)
         
         private void SpawnObject(Vector3 opticalSpawnPoint, Quaternion rotation)
         {
-            var spawnPoint = ObjectPlacementUtility.GetSpawnPoint();
+            var spawnPoint = ObjectPlacementUtility.GetSpawnPoint(); //TODO this should be replaced and unified with the new heightmap feature for default height instead of the 0 plane
```

Comment:
Kan je deze changes voor nu reverten? Dit conflicteert met mijn PR en is het flink aangepast 😅